python/lsst/meas/extensions/simpleShape/__init__.py

```diff
@@ -1,5 +1,7 @@
+from lsst.meas.base import BasePlugin, wrapSimpleAlgorithm
```

Comment:
I has simply copied the format from HSM, but this is a good point and is now updated